CHANGELOG

```diff
@@ -1,3 +1,5 @@
+- Add support to Reset App storage via API and Info Page
```

Comment:
I can add these at the end. If they get added early before release it can cause issues with other scripts. Just be sure the PR has a nice name and it will get added at release time.